src/modules/app/routeConfigs/extras.js

```diff
@@ -0,0 +1,54 @@
+// @flow
```

Comment:
The routeConfigs should be named like the classes:  e.g. ExtrasRouteConfig

Reply:
We also have to decide how we want to name our directories: eg. camelCase or kebab-case

As all npm packages use kebab-case I strongly suggest that we use it.

Reply:
Done